src/main/java/hudson/remoting/Launcher.java

```diff
@@ -111,7 +116,14 @@
     @Option(name="-ping")
     public boolean ping = true;
 
-    @Option(name="-slaveLog", usage="create local slave error log")
+    //TODO: rename the option
+    /**
+     * Specifies a destination for error logs.
+     * If specified, this option overrides the default destination within {@link #workDir}.
+     * If both this options and {@link #workDir} is not set, the log will not be generated.
+     */
+    @Option(name="-slaveLog", usage="Local agent error log destination (overrides workDir)")
```

Comment:
I was considering renaming.
My main concern about `agent` is that we will have `Jenkins agent` and `remoting agent`terms. And they won't be equal in general.